documentation/airport-guides/kric.md

```diff
@@ -0,0 +1,14 @@
+# KRIC - Richmond International Airport
+
+## About RIC
+Richmond International Airport (IATA: RIC, ICAO: KRIC, FAA LID: RIC) is a joint civil-military airport in Sandston, Virginia, United States, an unincorporated community (in Henrico County). The airport is about 7 miles (11 km) southeast of downtown Richmond, the capital of the Commonwealth of Virginia. Richmond International Airport is the busiest airport in central Virginia and the third-busiest in the state[citation needed] behind Washington D.C.'s two major airports, Washington Dulles and Washington National. RIC covers 2,500 acres (1,012 ha) of land.(Wikipedia)
+
+## Airspace - Potomac Tracon (James River / JRV Area)
+The PCT JRV sector extends from the surface to 12,000ft in most areas. For the purposes of Richmond, only the FLTRK ("Flat Rock") and TAPPA sectors are available. These sectors extend roughly 35 nautical miles in each direction from the airport. Within these sectors, there are a number of Restricted Areas and Military Operations Areas that aircraft must avoid. 
```

Comment:
Is there a reason the CSIDE and CSIDW sectors haven't been included?

Reply:
Looking again at the SOPs, the primary reason is that in my (probably 50 or more) times flying into RIC in real life, I've never been handed off to those frequencies. While they are technically part of Potomac, but don't really serve RIC directly.

Reply:
Good enough for me